src/models/modal/modal-open-request.v1.model.ts

```diff
@@ -1,5 +1,5 @@
 export type ModalSize = 'l' | 'm' | 's';
-export interface ModalOpenRequest {
+export interface ModalOpenRequestV1 {
```

Comment:
@zaheral This is a kind of breaking changes. It doesn't break functionality, but it will break the types.
This might be a good thing though, as it will make users of the API aware that we have released V2 of the MODAL.OPEN event.

Reply:
This would probably break the build-process of an app that is using this type. Apps who needs to be deployed would be blocked (e.g. Hotfix). I would keep it with "ModalOpenRequest" and the next version can be "ModalOpenRequestV2", so we don't block the people.

Reply:
Alright, sure. I will update the PR momentarily.